lib/models/menu_model.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+
+class MenuModel with ChangeNotifier{
+
+  int _itemSeleccionado = 0;
+
+  Color _backgroudColor = Colors.white;
+  Color _activeColor     = Colors.black;
+  Color _inactiveColor   = Colors.blueGrey;
+
+  int get itemSeleccionado => this._itemSeleccionado;
+
+  Color get backgroudColor => this._backgroudColor;
+  Color get activeColor => this._activeColor;
+  Color get inactiveColor => this._inactiveColor;
+
+  set itemSeleccionado (int index){
+    this._itemSeleccionado = index;
+    notifyListeners();
+  }
+
+  set backgroudColor (Color color){
```

Comment:
El código que se agrega, debe seguir el mismo formato que tiene la base de código que ya existe.

```suggestion
  set backgroudColor (Color color) {
```